roles/openshift_facts/defaults/main.yml

```diff
@@ -10,6 +10,9 @@ l_openshift_images_dict:
   origin: 'docker.io/openshift/origin-${component}:${version}'
   openshift-enterprise: 'registry.access.redhat.com/openshift3/ose-${component}:${version}'
 
+# oreg_url is defined by user input.
```

Comment:
Remind me again why you're opposed to setting a default for oreg_url here? To me it should just be defined with the exact same value of l_osm_registry_url_default below. One thing I may have poorly communicated is that we want to make sure that everything in openshift uses fully qualified image names by default and stop relying on additional-registries.

Reply:
It seems like we should handle that one as an unfortunate one off.
```
l_asb_default_images_dict:
  origin: 'docker.io/ansibleplaybookbundle/origin-ansible-service-broker:latest'
  openshift-enterprise: oreg_url
l_asb_default_images_default: "{{ l_asb_default_images_dict[openshift_deployment_type] }}"
l_asb_image_url: "{{ l_asb_default_images_default | regex_replace('${version}' | regex_escape, openshift_image_tag) }}"
ansible_service_broker_image: "{{ l_asb_image_url | regex_replace('${component}' | regex_escape, 'ansible-service-broker') }}"
```
${version} and ${component} aren't going to match for the origin case and that's intentional.